alerts.go

```diff
@@ -231,56 +238,168 @@ func formatCheckMessage(msg string) string {
 
 func doAlertsRetrieve(c *cli.Context) error {
 	client := newMackerelFromContext(c)
+	isClosed := c.Bool("withClosed")
+	limit := c.Int("limit")
+	count := 0
 
-	alerts, err := client.FindAlerts()
-	logger.DieIf(err)
-	PrettyPrintJSON(alerts)
+	if isClosed {
+		alerts, id, err := client.FindWithClosedAlerts()
+		logger.DieIf(err)
+		if id != "" {
+			for {
+				nextAlerts := alerts
```

Comment:
Does it work properly with >2 pages? (I think not)
Since `alerts` are being appended in L255, those 2 lines are unnecessary and L252 should be just `if limit >= len(alerts)`.